presto-spi/src/main/java/com/facebook/presto/spi/CatalogSchemaTableName.java

```diff
@@ -39,12 +36,17 @@ public static CatalogSchemaTableName valueOf(String catalogSchemaTableName)
 
     public CatalogSchemaTableName(String catalogName, SchemaTableName schemaTableName)
     {
+        checkNotEmpty(catalogName, "catalogName");
+        requireNonNull(schemaTableName, "schemaTableName is null");
         this.catalogName = catalogName;
         this.schemaTableName = schemaTableName;
     }
 
     public CatalogSchemaTableName(String catalogName, String schemaName, String tableName)
     {
+        checkNotEmpty(catalogName, "catalogName");
+        checkNotEmpty(schemaName, "schemaName");
```

Comment:
The checks for schema/table name are not necessary, since they are passed directly to the `SchemaTableName` constructor which already does them.